### PR TITLE
Add base url to require config dependencies

### PR DESCRIPTION
**Pull request by crouillon**
https://github.com/backbee/backbee-standard/issues/406

### CODE REVIEW

File: src/tb/apps/content/components/ContentManager.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ define(
             idDataAttribute: 'bb-id',
             droppableClass: '.bb-droppable',
             imageClass: 'Element/Image',
-            defaultPicturePath: '/resources/toolbar/html/img/filedrop.png',
+            defaultPicturePath: require.toUrl('html/img/filedrop.png'),
```
Comment (ndufreche) [inline src/tb/apps/content/components/ContentManager.js]: This fonction is require function ?

Comment (crouillon) [inline src/tb/apps/content/components/ContentManager.js]: yes, it resolves the URI by itself

Comment (ndufreche) [inline src/tb/apps/content/components/ContentManager.js]: ok

**Comment by ndufreche** [comment]
thanks :+1:

**Pull request merged**